adit/core/utils/dicom_to_nifti_converter.py

```diff
@@ -1,10 +1,36 @@
 import logging
 import subprocess
+from enum import IntEnum
 from pathlib import Path
 
+from adit.core.errors import (
+    DicomConversionError,
+    ExternalToolError,
+    InputDirectoryError,
+    InvalidDicomError,
+    NoValidDicomError,
+    OutputDirectoryError,
+)
+
 logger = logging.getLogger(__name__)
 
 
+class DcmExitCode(IntEnum):
```

Comment:
Can we rename this DcmToNiftiExitCode? Makes it more clear what exit code it is.